lib/routes/nintendo/eshop_us.js

```diff
@@ -1,36 +1,30 @@
+const { art } = require('@/utils/render');
 const got = require('@/utils/got');
+const path = require('path');
 
 module.exports = async (ctx) => {
     const response = await got({
         method: 'post',
-        url: `https://u3b6gr4ua3-dsn.algolia.net/1/indexes/*/queries?x-algolia-agent=Algolia%20for%20vanilla%20JavaScript%20(lite)%203.22.1%3BJS%20Helper%202.20.1&x-algolia-application-id=U3B6GR4UA3&x-algolia-api-key=9a20c93440cf63cf1a7008d75f7438bf`,
+        url: `https://u3b6gr4ua3-dsn.algolia.net/1/indexes/store_game_en_us_release_des/query?x-algolia-application-id=U3B6GR4UA3&x-algolia-api-key=a29c6927638bfd8cee23993e51e721c9`,
```

Comment:
```suggestion
        url: 'https://u3b6gr4ua3-dsn.algolia.net/1/indexes/store_game_en_us_release_des/query',
        headers: {
            'x-algolia-api-key': 'a29c6927638bfd8cee23993e51e721c9',
            'x-algolia-application-id': 'U3B6GR4UA3',
        },
```
These two params can be placed in the headers